docs/stcal/ramp_fitting/description.rst

```diff
@@ -317,3 +320,38 @@ that pixel will be flagged as JUMP_DET in the corresponding integration in the
 "rateints" product.  That pixel will also be flagged as JUMP_DET in the "rate"
 product.
 
+Likelihood Algorithm Details
+----------------------------
+As an alternative to the OLS algorithm, a likelihood algorithm can be selected
+with for ``--ramp_fitting.algorithm=LIKELY``.  If this algorithm is selected,
+the normal pipeline jump detection algorithm is skipped because this algorithm
+has its own jump detection algorithm.  The jump detection for this algorithm
+requires NGROUPS to be a minimum of four (4).  If NGROUPS :math:`\le` 3, then
+this algorithm is deselected, defaulting to the above described OLS algorithm
+and the normal jump detection pipeline step is run.
+
+Each pixel is independently processed, but rather than operate on the each
+group/resultant directly, the likelihood algorithm is based on differences of
+the groups/resultants :math:`d_i = r_i - r_{i-1}`.  The model used to determine
+the slope/countrate, :math:`a`, is:
+
+.. math::    
+    \chi^2 = ({\bf d} - a \cdot {\bf 1})^T C ({\bf d} - a \cdot {\bf 1}) \,,
+
+Differentiating, setting to zero, then solving for :math:`a` results in 
+
+.. math::    
+    a = ({\bf 1}^T C {\bf d})({\bf 1}^T C {\bf 1})^T \,,
+
+The covariance matrix :math:`C` is a tridiagonal matrix, due to the nature of the
+differences.  Because the covariance matrix is tridiagonal, the  computational
+complexity from :math:`O(n^3)` to :math:`O(n)`.  To see the detailed derivation
+and computations implemented, refer to 
+`Brandt (2024) <https://arxiv.org/abs/2309.08753>`_.  The Poisson and read noise 
```

Comment:
Update to published papers:
https://iopscience.iop.org/article/10.1088/1538-3873/ad38d9
https://iopscience.iop.org/article/10.1088/1538-3873/ad38da